bin/kafka-run-class.sh

```diff
@@ -245,7 +245,12 @@ GC_FILE_SUFFIX='-gc.log'
 GC_LOG_FILE_NAME=''
 if [ "x$GC_LOG_ENABLED" = "xtrue" ]; then
   GC_LOG_FILE_NAME=$DAEMON_NAME$GC_FILE_SUFFIX
-  KAFKA_GC_LOG_OPTS="-Xloggc:$LOG_DIR/$GC_LOG_FILE_NAME -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=100M"
+  JAVA_VERSION=$($JAVA -version 2>&1 | sed -E -n 's/.* version "([^.-]*).*"/\1/p')
+  if [[ "$JAVA_VERSION" -ge "9" ]] ; then
+    KAFKA_GC_LOG_OPTS+="-Xlog:gc*:$LOG_DIR/$GC_LOG_FILE_NAME:time,tags:filecount=10,filesize=102400"
```

Comment:
I didn't know about `uptime`. I feel `time` would be more useful to correlate with the server logs. We probably ended up with `-XX:+PrintGCTimeStamps` because `-XX:+PrintGCDateStamps` is not as widely known, but we can use `uptime` if we prefer our old behavior.